[2025-09-08T17:08:29.835+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-09-08T17:08:29.846+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cron_catchup_backfill.taskA backfill__2025-04-20T12:00:00+00:00 [queued]>
[2025-09-08T17:08:29.852+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cron_catchup_backfill.taskA backfill__2025-04-20T12:00:00+00:00 [queued]>
[2025-09-08T17:08:29.852+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-09-08T17:08:29.870+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): taskA> on 2025-04-20 12:00:00+00:00
[2025-09-08T17:08:29.875+0000] {standard_task_runner.py:72} INFO - Started process 19956 to run task
[2025-09-08T17:08:29.880+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'cron_catchup_backfill', 'taskA', 'backfill__2025-04-20T12:00:00+00:00', '--job-id', '127', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/executing_cron_catchup_backfill.py', '--cfg-path', '/tmp/tmpjzqpvc63']
[2025-09-08T17:08:29.881+0000] {standard_task_runner.py:105} INFO - Job 127: Subtask taskA
[2025-09-08T17:08:29.914+0000] {task_command.py:467} INFO - Running <TaskInstance: cron_catchup_backfill.taskA backfill__2025-04-20T12:00:00+00:00 [running]> on host codespaces-9bd7b3
[2025-09-08T17:08:29.957+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='loonycorn' AIRFLOW_CTX_DAG_ID='cron_catchup_backfill' AIRFLOW_CTX_TASK_ID='taskA' AIRFLOW_CTX_EXECUTION_DATE='2025-04-20T12:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='backfill__2025-04-20T12:00:00+00:00'
[2025-09-08T17:08:29.957+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-09-08T17:08:29.968+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-09-08T17:08:29.968+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo TASK A has executed!']
[2025-09-08T17:08:29.975+0000] {subprocess.py:86} INFO - Output:
[2025-09-08T17:08:29.976+0000] {subprocess.py:93} INFO - TASK A has executed!
[2025-09-08T17:08:29.976+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-09-08T17:08:29.990+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-09-08T17:08:29.990+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=cron_catchup_backfill, task_id=taskA, run_id=backfill__2025-04-20T12:00:00+00:00, execution_date=20250420T120000, start_date=20250908T170829, end_date=20250908T170829
[2025-09-08T17:08:30.015+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-09-08T17:08:30.033+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-09-08T17:08:30.037+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
